### DISCUSSION TITLE
Paperless-NGX 2.11.0 Update ERROR

### DISCUSSION THREAD

**Discussion by nicedevil007** [tteck/Proxmox#3362] [Help Needed] [answered]
I get this error:
![image](https://github.com/tteck/Proxmox/assets/17103076/1891b8e7-3bd9-4564-8e12-49d17261524a)

If I do it manually, I get this error:
![image](https://github.com/tteck/Proxmox/assets/17103076/0c9f290d-9571-4c1f-8d8b-14f036981b86)

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/discussions/3358
